Example/Authenticator/Models/Database/Connection.swift

```diff
@@ -55,7 +55,7 @@ enum ConnectionStatus: String {
         case name
         case code
         case baseUrlString = "connect_url"
-        case logoUrlString = "provier_logo_url"
+        case logoUrlString = "provider_logo_url"
```

Comment:
Also in `SeProviderResponseV2` change logoUrl value from `logo_url` to `provider_logo_url`